### DISCUSSION TITLE
Where to add my credentials for opensubtitles

### DISCUSSION THREAD

**Discussion by jerrydelivo** [smplayer-dev/smplayer#251] [Q&A]
I am using smplayer version 19.10.2 . It says "Login to opensubtitles.org has failed" every time I try to download subtitles. The thing is... I can't find where I put my credentials (username, password) for opensubtitles.org . I remember in the past I was using smplayer and I added somewhere my credentials and it was  fine from then on. I go subtitles -> find subtitles online -> options... there is no form to add my credentials. Even when I go to preferences-> subtitles I can't find it... Any ideas?

**Comment by smplayer-dev** [comment]
The option to enter your credentials was added in version 20.4, so you need to update. It's in subtitles -> find subtitles -> options.

**Reply by jerrydelivo** [reply]
Yeah for some reason "sudo apt-get install smplayer" on ubuntu finds version 19.10.2 to be the latest released. Didn't know I have to install it with snap. Anyways fixed it. Thanks

**Reply by smplayer-dev** [reply]
This should install the latest version from the smplayer repository:
```
sudo add-apt-repository ppa:rvm/smplayer 
sudo apt-get update 
sudo apt-get install smplayer 
```

You can also find deb packages here: https://github.com/smplayer-dev/smplayer/releases
